52/abkosar/timer.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+import time
+import os
+from datetime import datetime, timedelta
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Pomodoro Timer')
+    parser.add_argument('--duration',
+                        '-d',
+                        dest='duration',
+                        type=int,
+                        help='Duration of the timer in minutes',
+                        required=True)
+
+    args = parser.parse_args()
+    timer_duration = args.duration
+    timer = timedelta(minutes=timer_duration)
+    one_second = timedelta(seconds=1)
+
+    i = 0
+    print(timer.seconds)
+    while i < timer.seconds:
```

Comment:
You might consider using something like a for loop and the range() built-in function here in order for your code to be more pythonic,

Reply:
Should I fix these and do another PR?